tests/spark/test_mrjob_spark_harness.py

```diff
@@ -53,6 +54,30 @@ class MRSortAndGroupReversedText(MRSortAndGroup):
     INTERNAL_PROTOCOL = ReversedTextProtocol
 
 
+class MRPassThruArgTest(MRJob):
+
+    INTERNAL_PROTOCOL = ReversedTextProtocol
```

Comment:
Unless `ReversedTextProtocol` is important to testing passthru args, you should just use the default protocols.

Reply:
Nope, it is not important. I will just remove this